target_postgres/__init__.py

```diff
@@ -93,7 +105,8 @@ def persist_lines(config, lines):
                 raise Exception("Line is missing required key 'stream': {}".format(line))
             stream = o['stream']
             schemas[stream] = o
-            validators[stream] = Draft4Validator(o['schema'])
+            schema = float_to_decimal(o['schema'])
+            validators[stream] = Draft4Validator(schema, format_checker=FormatChecker())
```

Comment:
@koszti why do we specifically need a `FormatChecker` now?

Reply:
@micaelbergeron It doesn't look like it's actually necessary, since it concerns the `format` property and has nothing to do with decimals or `multipleOf`, but I don't think it hurts to keep it either.

Koszti may just have accidentally included two unrelated fixes at the same time when he submitted https://github.com/statsbotco/target-postgres/pull/11. This `FormatChecker` is still present in https://github.com/transferwise/pipelinewise-target-postgres/blob/master/target_postgres/__init__.py#L136, so we're probably OK keeping it here too. 

I'm more than happy to submit a new PR that doesn't apply this change, though, if you prefer.